docs/_snippets/medical-record-status.mdx

```diff
@@ -0,0 +1,10 @@
+  <Expandable title="properties">
+    <ResponseField name="exists" type="boolean" required>
+      Whether the rendering has been generated.
+    </ResponseField>
+
+    <ResponseField name="createdAt" type="string">
+      When the rendering was created.
```

Comment:
we call this a medical record summary, not rendering -> suggest to change "rendering" to "summary"